.github/workflows/main.yml

```diff
@@ -1,7 +1,12 @@
 name: All the CI
 
-on: [push, pull_request]
-
+on:
+  push:
+    branches:
+      - master
+  pull_request:
```

Comment:
Can be shortened to pull_request: {}

Reply:
Just curious, do you have any links to the docs for that? I looked everywhere for them on a different project but couldn't find 'em!

Reply:
Nothing but https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#on
It is implicit, it should get the same result as
```
on:
  - pull_requests
```
Or, to be precise you can do :
```
on:
  pull_request:
    types:
      - created
      - synchronize
```

Reply:
In fact, could you please specify events like I did in my second sample?

Reply:
Do we want to use that exact list, or anything additionally?

Reply:
I was almost sure I did not pushed the comment button in here :o.
I did some tests by renaming the PR (trigger the `edited` event), actions were not re-triggered, so I guess the defaults for `pull_request` already excludes theses configs and thus already fits our need!